tools/build.sh

```diff
@@ -4,7 +4,6 @@
 
 set -e
 
-build_clean=
 build_root=$(cd "$(dirname "$0")/.." && pwd)
```

Comment:
>d_r [](start = 4, length = 3)

It would actually be so nice if this script would die a horrible death.
I think all the options handled by this script can be directly passed to cmake, isn't it. It seems this script is an extra abstraction layer that is not really necessary ...